### PR TITLE
SQL-scripts for SignUp

### LINKED ISSUE #34: SQL scripts for SignUp Flow
IMS needs a database to store user's data. As a database we use a PostgreSQL. 
The script called `SignUp_init.up.sql` creates following tables and columns:
1. `users`:          (information about user)
     - `id`;
     - `last_name`;
     - `first_name`;
     - `email`;
     - `verified`;          (shows user's verification status)
2. `credentials`:          (holds user's password hash)
     - `id`;
     - `password_hash`;
     - `salt`;
     - `updated_at`;          (shows the last time when password was updated)
3. `email_verification_tokens`:          (holds user's verification token)
     - `id`;
     - `verification_token`;
     - `generated_at`.          (shows the last time when token was generated)

The `id` column is a primary key. 
Relationships between tables:
1. One-to-one relationship:
     - `users` <-> `credentials`;
     - `users` <-> `email_verification_tokens`.

If you want to delete all information from the database you can use this script - `SignUp_init.down.sql`.

### PR DESCRIPTION
**Pull request by Krimyllex**
Closes #34

### CODE REVIEW

File: pkg/signup/repository/postgresql/SignUp_init.down.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+BEGIN;
+
+DROP TABLE IF EXISTS users CASCADE;
+DROP TABLE IF EXISTS emailverificationtokens CASCADE;
```
Comment (taaraora) [inline pkg/signup/repository/postgresql/SignUp_init.down.sql:4]: `emailverificationtokens` -> `email_verification_tokens`

File: pkg/signup/repository/postgresql/SignUp_init.up.sql [hunk 2]
```diff
@@ -0,0 +1,24 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS users (
+    id int PRIMARY KEY,
```
Comment (taaraora) [inline pkg/signup/repository/postgresql/SignUp_init.up.sql:4]: is should be serial

File: pkg/signup/repository/postgresql/SignUp_init.up.sql [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    first_name varchar(255) NOT NULL,
+    last_name varchar(255) NOT NULL,
+	email varchar(255) NOT NULL,
+	verified boolean default false NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS email_verification_tokens (
+    id int PRIMARY KEY REFERENCES users(id),
```
Comment (taaraora) [inline pkg/signup/repository/postgresql/SignUp_init.up.sql:12]: all keywords should be uppercase

File: pkg/signup/repository/postgresql/SignUp_init.up.sql [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    verification_token varchar(255) NOT NULL,
+	generated_at date NOT NULL
```
Comment (taaraora) [inline pkg/signup/repository/postgresql/SignUp_init.up.sql:14]: please change the type to `timestamp without time zone`

**Comment by Krimyllex** [comment]
I created new path to SQL scripts for comfortable usage. Old path with old scripts I deleted.

**Pull request merged**